test/decorators/product_decorator_test.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+require 'active_decorator_test_case'
+
+class ProductDecoratorTest < ActiveDecoratorTestCase
+  test 'practice_content_for_toggle' do
```

Comment:
```suggestion
  test '#practice_content_for_toggle' do
```

instanceメソッドなのでこの表記になるはずです〜